app/layout.tsx

```diff
@@ -27,32 +18,6 @@ export default function RootLayout(
     return (
         <html lang="en">
             <body className={inter.className}>
-                <nav className="flex h-[80px] w-full items-center justify-around bg-primary-600 md:content-center">
```

Comment:
you can leave the navbar in the layout, since all other pages will re-use it too